integration-tests/integration/appointments/addAppointment.spec.js

```diff
@@ -91,6 +91,7 @@ context('A user can add an appointment', () => {
     const addAppointmentPage = AddAppointmentPage.verifyOnPage()
     const form = addAppointmentPage.form()
     form.date().type(moment().format('DD/MM/yyyy'))
+    addAppointmentPage.activeDate().click()
```

Comment:
Sometimes the datepicker doesn't go away when it's supposed to, covering other fields. We've used this approach successfully in bulk and court appointment, so just doing here as well.